server/controllers/Users.js

```diff
@@ -86,6 +86,24 @@ class Users {
       return serverError(res);
     }
   }
+
+  /**
+   * @name changePassword
+   * @async
+   * @static
+   * @memberof Users
+   * @param {Object} req express request object
+   * @param {Object} res express response object
+   * @returns {JSON} JSON object with details of new user
+   */
+  static async changePassword(req, res) {
```

Comment:
where is the check to ensure `password` and `confirm_password` match

Reply:
it was handled in the middleware server/middlewares/passwordValidation.js